README.md

```diff
@@ -126,6 +122,11 @@ exports.handler = (event, context, callback) => {
     smartapp.handleLambdaCallback(event, context, callback);
 };
 ```
+There are also a few Glitch examples:
+
+- [Simple SmartThings Automation App using Contact Sensors](https://glitch.com/edit/#!/south-mayonnaise?path=README.md:1:0)
+- [Simple SmartThings Automation App using Motion Detectors](https://glitch.com/edit/#!/polite-math?path=README.md:1:0)
+- [Simple Switch C2C App](https://glitch.com/edit/#!/aquamarine-crop?path=README.md:1:0)
```

Comment:
Can you amend this copy to be more friendly to ST-beginners?  I'm thinking we spell out that abbreviation: `Simple Switch Cloud-to-Cloud (C2C) App`